cli/cli.go

```diff
@@ -433,9 +443,13 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 			Flags: []cli.Flag{
 				cli.BoolFlag{Name: "authcode, a", Usage: "use browser to authenticate via oauth2 authorization code grant"},
 				cli.BoolFlag{Name: "client, c", Usage: "authenticate with oauth2 client ID and secret"},
+				cli.StringFlag{Name: "identifier, i", Usage: "Override client identifier, default is " + cliClientID},
```

Comment:
minor: maybe "id" is enough instead of the full "identifier" since we always talk about client id.